data.sql

```diff
@@ -1,5 +1,13 @@
 /* Populate database with sample data. */
 
-INSERT INTO animals (name) VALUES ('Luna');
-INSERT INTO animals (name) VALUES ('Daisy');
-INSERT INTO animals (name) VALUES ('Charlie');
+INSERT INTO animal (id, name,date_of_birth,escape_attempts,neutered,weight_kg)
+VALUES (1,'Agmon',DATE '2020-02-03',0,true,10.23);
```

Comment:
- [ ] I think you spelled Agumon wrongly while inserting your data. Kindly check it out 👍🏽